app/views/applications/index.html.erb

```diff
@@ -3,7 +3,7 @@
 
   <% if @application.present?  %>
     <p><%= current_user.last_name %>, <%= current_user.first_name %> - <%= link_to 'View application', "/applications/#{@application.id}"%></p>
+  <% elsif %>
```

Comment:
So you have an `if` here with no condition. If there is nothing to check, you can just do `else`